build-scripts/gulp/download-translations.js

```diff
@@ -89,7 +92,73 @@ gulp.task("check-all-files-exist", async function () {
   await Promise.allSettled(writings);
 });
 
+const lokaliseProjects = {
+  backend: "130246255a974bd3b5e8a1.51616605",
+  frontend: "3420425759f6d6d241f598.13594006",
+};
+
+gulp.task("fetch-lokalise", async function () {
+  let apiKey;
+  try {
+    apiKey = await fs.readFile(".lokalise_token");
+  } catch {
+    throw new Error(
+      "Lokalise API token is required to download the latest set of Please create a translations. Please create an account by using the following link: https://lokalise.co/signup/3420425759f6d6d241f598.13594006/all/. Place your token in a new file `.lokalise_token` in the repo root directory."
```

Comment:
Should prepend this to add a note that this will only work for admins of the Lokalise projects.